packages/cactus-plugin-bungee/README.md

```diff
@@ -0,0 +1,159 @@
+# `@hyperledger/cactus-plugin-bungee`
+
+ The package provides `Hyperledger Cacti` a way to create blockchain snapshots and views for different distributed ledgers currently supported by Cacti. The implementation follows the paper [BUNGEE](https://dl.acm.org/doi/pdf/10.1145/3643689)
```

Comment:
@eduv09 The abstract of the paper mentions what BUNGEE stands for and I'd love it if you included that here as well just so that people get a quicker understanding of it from the full expression.

Reply:
@eduv09 I see that you resolved this. Could you point out where in the README the acronym's full meaning is shown?

Reply:
![image](https://github.com/hyperledger/cacti/assets/61276055/ec3f05d2-9dca-4197-8882-76f7cfe12a8e)
![image](https://github.com/hyperledger/cacti/assets/61276055/000b8a43-6848-438f-94e6-f3232c4a3120)


The changes were introduces in commit e319e6c, if I am not mistaken.